--- a/include/image.mk
+++ b/include/image.mk
@@ -609,6 +609,7 @@ define Device/Build/image
 	IMAGE_TYPE=$(word 1,$(subst ., ,$(2))) \
 	IMAGE_FILESYSTEM="$(1)" \
 	IMAGE_PREFIX="$(IMAGE_PREFIX)" \
+	KERNEL_INITRAMFS="$(KERNEL_INITRAMFS)" \
 	DEVICE_VENDOR="$(DEVICE_VENDOR)" \
 	DEVICE_MODEL="$(DEVICE_MODEL)" \
 	DEVICE_VARIANT="$(DEVICE_VARIANT)" \

--- a/scripts/json_add_image_info.py
+++ b/scripts/json_add_image_info.py
@@ -48,6 +48,7 @@ def get_titles():
     "profiles": {
         device_id: {
             "image_prefix": getenv("IMAGE_PREFIX"),
+            "kernel_initramfs": getenv("KERNEL_INITRAMFS"),
             "images": [
                 {
                     "type": getenv("IMAGE_TYPE"),